src/components/fabSpeedDial/fabSpeedDial.scss

```diff
@@ -0,0 +1,167 @@
+        transition: $custom-transition;
+      }
+    }
+  }
+
+  /*
+   * Handle the animations
+   */
+  &.scale {
+    .md-fab-action-item {
+      transform: scale(0);
+      transition: $custom-transition;
+    }
+  }
+
+  &.fling {
+    &.md-down {
+      .md-fab-action-item {
+        &:nth-child(1) { transform: translateY(-$trigger-height * 1 - $trigger-offset); z-index: 6; }
+        &:nth-child(2) { transform: translateY(-$trigger-height * 2 - $trigger-offset); z-index: 5; }
+        &:nth-child(3) { transform: translateY(-$trigger-height * 3 - $trigger-offset); z-index: 4; }
+        &:nth-child(4) { transform: translateY(-$trigger-height * 4 - $trigger-offset); z-index: 3; }
+        &:nth-child(5) { transform: translateY(-$trigger-height * 5 - $trigger-offset); z-index: 2; }
+        &:nth-child(6) { transform: translateY(-$trigger-height * 6 - $trigger-offset); z-index: 1; }
+      }
+    }
+
+    &.md-up {
+      .md-fab-action-item {
+        &:nth-child(1) { transform: translateY($trigger-height * 1 + $trigger-offset); z-index: 6; }
+        &:nth-child(2) { transform: translateY($trigger-height * 2 + $trigger-offset); z-index: 5; }
+        &:nth-child(3) { transform: translateY($trigger-height * 3 + $trigger-offset); z-index: 4; }
+        &:nth-child(4) { transform: translateY($trigger-height * 4 + $trigger-offset); z-index: 3; }
+        &:nth-child(5) { transform: translateY($trigger-height * 5 + $trigger-offset); z-index: 2; }
+        &:nth-child(6) { transform: translateY($trigger-height * 6 + $trigger-offset); z-index: 1; }
+      }
+    }
+
+    &.md-left {
+      .md-fab-action-item {
+        &:nth-child(1) { transform: translateX($trigger-height * 1 + $trigger-offset + 5); z-index: 6; }
+        &:nth-child(2) { transform: translateX($trigger-height * 2 + $trigger-offset + 5); z-index: 5; }
+        &:nth-child(3) { transform: translateX($trigger-height * 3 + $trigger-offset + 5); z-index: 4; }
+        &:nth-child(4) { transform: translateX($trigger-height * 4 + $trigger-offset + 5); z-index: 3; }
+        &:nth-child(5) { transform: translateX($trigger-height * 5 + $trigger-offset + 5); z-index: 2; }
+        &:nth-child(6) { transform: translateX($trigger-height * 6 + $trigger-offset + 5); z-index: 1; }
+      }
+    }
+
+    &.md-right {
+      .md-fab-action-item {
+        &:nth-child(1) { transform: translateX(-$trigger-height * 1 - $trigger-offset - 5); z-index: 6; }
+        &:nth-child(2) { transform: translateX(-$trigger-height * 2 - $trigger-offset - 5); z-index: 5; }
+        &:nth-child(3) { transform: translateX(-$trigger-height * 3 - $trigger-offset - 5); z-index: 4; }
+        &:nth-child(4) { transform: translateX(-$trigger-height * 4 - $trigger-offset - 5); z-index: 3; }
+        &:nth-child(5) { transform: translateX(-$trigger-height * 5 - $trigger-offset - 5); z-index: 2; }
+        &:nth-child(6) { transform: translateX(-$trigger-height * 6 - $trigger-offset - 5); z-index: 1; }
+      }
```

Comment:
Using JS would also allow us to calculate the trigger height rather than specifying it in the SCSS. Could mean supporting more/different/custom FABs. I'll see what I can drum up.